docs/boards/nrf52840_dongle.md

```diff
@@ -0,0 +1,73 @@
+# <img alt="OpenSK logo" src="../img/OpenSK.svg" width="200px">
+
+## Nordic nRF52840 Dongle
+
+![Nordic dongle](../img/dongle_front.jpg)
+
+### Flashing using DFU
+
+To flash the firmware, run:
+
+```shell
+./deploy.py --board=nrf52840_dongle_dfu --opensk --programmer=nordicdfu
+```
+
+The script will ask you to switch to DFU mode. To activate that on your dongle,
+keep the button pressed while inserting the device into your USB port. You may
+additionally need to press the tiny, sideways facing reset button. The device
+indicates DFU mode with a slowly blinking red LED.
+
+### Flashing with an external programmer (JLink, OpenOCD, etc.)
+
+If you want to use JTAG with the dongle, you need additional hardware.
+
+*   a [Segger J-Link](https://www.segger.com/products/debug-probes/j-link/) JTAG
+    probe.
+*   a
+    [TC2050 Tag-Connect programming cable](https://www.tag-connect.com/product/tc2050-idc-nl-10-pin-no-legs-cable-with-ribbon-connector).
+*   a [Tag-Connect TC2050 ARM2010](http://www.tag-connect.com/TC2050-ARM2010)
+    adaptor
+*   optionally a
+    [Tag-Connect TC2050 retainer clip](http://www.tag-connect.com/TC2050-CLIP)
+    to keep the spring loaded connector pressed to the PCB.
+    
+Follow these steps:
+
+1.  The JTAG probe used for programming won't provide power to the board.
+    Therefore you will need to use a USB-A extension cable to power the dongle
+    through its USB port.
+
+1.  Connect the TC2050 cable to the pads below the PCB:
+
+    ![Nordic dongle pads](../img/dongle_pads.jpg)
+
+1.  You can use the retainer clip if you have one to avoid maintaining pressure
+    between the board and the cable:
+
+    ![Nordic dongle retainer clip](../img/dongle_clip.jpg)
+
+1.  Depending on the programmer you're using, you may have to adapt the next
+    command line. Run our script for compiling/flashing Tock OS on your device
+    (_output may differ_):
+
+    ```shell
+    $ ./deploy.py os --board=nrf52840_dongle_opensk --programmer=jlink
+    ```
+
+1.  Remove the programming cable and the USB-A extension cable.
+
```

Comment:
We're loosing the information about the 3D printed enclosure that has been designed for the dongle (this PR removes it from the `README.md` file)

Reply:
I really think this is a cool feature of OpenSK, so I should definitely mention it in the main README. The content is moved to the board definition. Would that work?